.github/workflows/sync-labels-npm.yml

```
@@ -0,0 +1,154 @@
+# Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/sync-labels-npm.md
+name: Sync Labels
+
+env:
+  # See: https://github.com/actions/setup-node/#readme
+  NODE_VERSION: 16.x
```

```suggestion
  NODE_VERSION: 12.x
```

There is some inconsistency (the test workflow uses 10.x), but during a workflow run using the action, it is executed using Node.js 12:

https://github.com/arduino/setup-protoc/blob/cd05e7c18944a80b6cf2d3e608d9ba8a7e6a2ded/action.yml#L15

Even though it is not so important for this specific piece of infrastructure, which is not likely to ever be used outside of the CI system, I still think it is best to use a single Node.js version across the entire project.